channels-rke2.yaml

```diff
@@ -8,7 +8,7 @@ appDefaults:
         defaultVersion: '1.22.x'
       - appVersion: '>= 2.6.5-0 < 2.6.7-0'
         defaultVersion: '1.23.x'
-      - appVersion: '>= 2.6.7-0 < 2.7.100-0'
+      - appVersion: '>= 2.6.7-0 < 2.7.2-0'
```

Comment:
I think this should be fine as the value is still greater than the current maximum Rancher version of 2.7.1. This should help with potential future merge conflicts, as this change was made to enable v1.25.x to be be `>= 2.7.2-0 < 2.7.100-0`, but calling it out here in this review just to be sure.